README.md

```diff
@@ -7,6 +7,39 @@ Doing fast searching of nearest neighbors in high dimensional spaces is an incre
 
 This project contains some tools to benchmark various implementations of approximate nearest neighbor (ANN) search for different metrics. We have pregenerated datasets (in HDF5) formats and we also have Docker containers for each algorithm. There's a [test suite](https://travis-ci.org/erikbern/ann-benchmarks) that makes sure every algorithm works.
 
+
```

Comment:
Can you please remove this, since (a) I don't think it belongs in the README (b) we use Docker for everything, not Conda

Reply:
Absolutely I'll remove this. It's only temporary to help others (and myself) during evaluation.